sdk/spring/azure-spring-boot-test-aad/src/test/java/com/azure/test/aad/group/AADAuthenticationFilterIT.java

```diff
@@ -121,14 +121,14 @@ private void testAADAuthenticationFilter(String clientId, String clientSecret, S
     @EnableGlobalMethodSecurity(securedEnabled = true, prePostEnabled = true)
     @SpringBootApplication
     @RestController
-    public static class DumbApp extends WebSecurityConfigurerAdapter {
+    public static class DumbApp extends AzureOAuth2Configuration {
```

Comment:
For `AADAuthenticationFilter`, we do not need to use `AzureOAuth2Configuration`.